plugin/aws/s3Object.go

```diff
@@ -23,14 +22,29 @@ type s3Object struct {
 	client *s3Client.S3
 }
 
-func newS3Object(name string, bucket string, key string, client *s3Client.S3) *s3Object {
+func newS3Object(rawObj *s3Client.Object, name string, bucket string, key string, client *s3Client.S3) *s3Object {
 	o := &s3Object{
 		EntryBase: plugin.NewEntry(name),
 		bucket:    bucket,
 		key:       key,
 		client:    client,
 	}
-	o.DisableCachingFor(plugin.Metadata)
+	o.DisableCachingFor(plugin.MetadataOp)
+
+	attr := plugin.EntryAttributes{}
+	mtime := awsSDK.TimeValue(rawObj.LastModified)
+	attr.SetCtime(mtime)
```

Comment:
We have no way to get creation time? Or will it never differ in S3? Could use a comment clarifying why it's LastModified given that these are very file-like.